elisp-bytecode.texi

```diff
@@ -1090,26 +1160,44 @@ Call @code{concat} with three arguments.
 Call @code{concat} with four arguments.
 
 
+@node concatN
+@unnumberedsubsubsec @code{concatN} (174)
+@kindex concatN
+
+Call @code{concat} an arbitrary number of arguments.  Note there are
```

Comment:
in the 0-255 range, concat with more arguments will generate a function call.

Reply:
That makes more sense of the generated code. Have fixed and in the opcode table it will be pushed as soon as I go over the rest.